codespell_lib/data/dictionary_rare.txt

```diff
@@ -69,7 +69,6 @@ deffer->differ, defer,
 degrate->degrade
 degrates->degrades
 dependant->dependent
-derails->details
```

Comment:
This is here for a reason. 'r' can be a typo of 't'.

Reply:
Indeed. Is this incompatible with `defails->details, derails,` in the main dictionary?

Also, I'm wondering why the singular was missing.

Reply:
This entry doesn't conflict with `defails->details, derails,` in the main dictionary.
I deleted this entry because most uses of `derails` on GitHub were correct.

What should I do with this entry?

Reply:
@luzpaz What do you think? Should we keep `derails->details`, and if so add `derail->detail`?

Reply:
I added this entry back.